pyfstat/make_sfts.py

```diff
@@ -97,12 +97,12 @@ def __init__(
         self.tbounds = [self.tstart, self.tend]
         logging.info("Using segment boundaries {}".format(self.tbounds))
 
-    def _get_setup_from_input_data(self):
+    def _get_setup_from_tstart_duration(self):
         self.tstart = int(self.tstart)
         self.duration = int(self.duration)
-        self.tend = self.tstart + self.duration
+        self.tend = self.tstart + self.duration + self.Tsft
```

Comment:
Now I'm getting a bit confused myself, but I think here it must be just
```suggestion
        self.tend = self.tstart + self.duration
```
without the extra Tsft because the `tstart` and `duration` come straight from human input who would e.g. say `duration=86400` if they actually mean 1 day and not 23.5 hours.

Reply:
Ok, that makes sense